components/patient-data/ui/src/main/resources/PhenoTips/PatientConsentConfiguration.xml

```diff
@@ -511,7 +511,24 @@ $xwiki.jsx.use($displayDocName)##
     }
   };
 
+  var __consent_enableRequired = function(trigger, toggleTarget) {
+    if (trigger.selectedIndex &gt;= 0 &amp;&amp; trigger.options[trigger.selectedIndex].value == '1') {
+      toggleTarget.disabled = false;
+    } else {
+      toggleTarget.disabled = true;
+    }
+  };
+
   var init = function() {
+    $$('select[name$="required"]').each(function(elt) {
+      var toggleTarget = elt.up('tr') &amp;&amp; elt.up('tr').down('td.affectsFields select');
+      if (!toggleTarget) return;
```

Comment:
Codestyle: please use `{}` even for single line blocks, just as we do in Java.

    if (!toggleTarget) {
      return;
    }